AGInputControls/Source/FloatingLabelTextField.swift

```diff
@@ -82,6 +82,24 @@ open class FloatingLabelTextField : FormattingTextField {
         }
     }
     
+    open var isRequired: Bool = false {
+        didSet {
+            configureColors()
+        }
+    }
+    
+    open var requiredText: String? = " *" {
```

Comment:
I think we should manage that space internally. Its not really obvious for user that he/she should take care of it

Reply:
moved to private property